src/glmtools.jl

```diff
@@ -14,8 +14,12 @@ linkinv(::CauchitLink, η) = oftype(η, 0.5) + atan(η) / pi
 mueta(::CauchitLink, η) = one(η) / (pi * (one(η) + abs2(η)))
 
 linkfun(::CloglogLink, μ) = log(-log1p(-μ))
-linkinv(::CloglogLink, η) = -expm1(-exp(η))
-mueta(::CloglogLink, η) = exp(η) * exp(-exp(η))
+function linkinv(::CloglogLink, η::T) where {T<:AbstractFloat}
```

Comment:
The `where` syntax for type parameters is only available in Julia 0.6 and later, but this package still supports 0.5. To continue to support 0.5, this will have to be
```julia
function linkinv{T<:AbstractFloat}(::CloglogLink, η::T)
```